crates/autopilot/src/database.rs

```diff
@@ -40,12 +50,32 @@ async fn count_rows_in_table(ex: &mut PgConnection, table: &str) -> sqlx::Result
     sqlx::query_scalar(&query).fetch_one(ex).await
 }
 
+async fn count_unused_app_data(ex: &mut PgConnection) -> sqlx::Result<i64> {
+    let query = r#"
+        SELECT
+            COUNT(*)
+        FROM app_data AS a
+        LEFT JOIN orders o
+            ON a.contract_app_data = o.app_data
```

Comment:
Is this going to work with LEFT JOIN or does it need LEFT OUTER JOIN?

Reply:
They are equivalent AFAICT. From the docs (emphasis my own):

> The words `INNER` and `OUTER` are optional in all forms. `INNER` is the default; `LEFT`, `RIGHT`, and `FULL` **imply an outer join**.

Also, as far as I can tell by quickly Googling, `LEFT JOIN` is equivalent to `LEFT OUTER JOIN`.